workspaces/default/themes/base/assets/styles/swagger-ui-kong-theme.css

```diff
@@ -134,6 +134,7 @@
   overflow: hidden;
 }
 .sidebar-list ul li .submenu-items a {
+  color: #1456cb;
```

Comment:
I know this color value was hard-coded before, but I think we should use a CSS variable for this color, like:
```css
:root {
  --link-color: #1456cb;
}
```